cli_tools/common/utils/storage/storage_client.go

```diff
@@ -34,8 +34,7 @@ import (
 
 var (
 	bucketNameRegex = `([a-z0-9][-_.a-z0-9]*)`
-	bucketPathRegex = regexp.MustCompile(fmt.Sprintf(`^gs://%s/(.*)$`, bucketNameRegex))
-	gsPathRegex     = regexp.MustCompile(fmt.Sprintf(`^gs://%s/(.+)$`, bucketNameRegex))
+	gsPathRegex     = regexp.MustCompile(fmt.Sprintf(`^gs://%s/?(.*)$`, bucketNameRegex))
```

Comment:
How does your regular expression handle `gs://bucket||||`? Is it what you expected?

Reply:
Good point. Made the slash between bucket name and object path a part of bucket matching group and removed it (if any) from the resulting string.

Reply:
Actually, this one is still not working as expected. Will fix it tomorrow.

Reply:
Actually handled it now